### ISSUE TITLE
Can someone provide an example of config for a regression?

### ISSUE THREAD

**Issue by pkundurthy** [amzn/amazon-dsstne#134]
I'm trying to understand how to setup a regression problem with dsstne. Can you provide an example?

I'm used to dealing with regression problems in Keras-- where you just don't specify an activation function for the output layer so that it returns the continuous output variable. Is it similar for dsstne? A config example would be helpfull. 

Thanks.

**Comment by scottlegrand** [comment]
Yes, use linear as the activation function and L2 as the error function.
